app/models/supplejack_api/user_set.rb

```diff
@@ -102,16 +102,33 @@ def name=(name)
     def self.custom_find(id)
       mongo_object_id_char_length = 24
 
-      user_set = if id.to_s.length == mongo_object_id_char_length
-                   find(id) rescue nil
-                 else
-                   where(url: id).first
-                 end
+      if id.to_s.length == mongo_object_id_char_length
+        find(id) rescue nil
+      else
+        where(url: id).first
+      end
     end
     # rubocop:enable Lint/UselessAssignment
 
     def self.all_public_sets
-      where(privacy: 'public', :name.ne => 'Favourites').order_by(updated_at: :desc)
+      where(privacy: 'public', :name.ne => 'Favourites').order(updated_at: :desc)
+    end
+
+    def self.search(page=1, per_page=10, attr_order=:updated_at, direction=:desc, search_term=nil)
```

Comment:
Add space here `page = 1, per_page = 10...` Rubocop should pick this one?

Reply:
Yes I made a new commit with rubocop changes.